.github/ISSUE_TEMPLATE/release-a-new-version.md

```diff
@@ -25,7 +25,7 @@ assignees: ''
 - [ ] Check the docker image of the release automatically published to [Docker Hub](https://hub.docker.com/r/stellar/anchor-platform).
 - [ ] If necessary, update the badges versions in [docs/00 - Stellar Anchor Platform.md].
 - [ ] Create the pull request `release/2.0.x -> main`: this should require two approvals. DO NOT squash merge.   
-- [ ] Create another pull request `main -> develop`: AFTER the release branch is merged with the `main` branch. DO NOT squash merge.
+- [ ] Create another pull request `release/2.0.x -> develop`: AFTER the release branch is merged with the `main` branch. DO NOT squash merge.
```

Comment:
Won't it have the same issue we had before, where main has a merge commit that develop doesn't have?

Reply:
We squashed merged to `main` in the previous release which caused the conflict.

Reply:
So we don't squash merge to main anymore right?

Reply:
I think squash merge to `main` is troublesome.